raster/r.his/testsuite/test_r_his.py

```diff
@@ -0,0 +1,90 @@
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+
+
+class TestRHis(TestCase):
+    hue = "elevation"
+    intensity = "elevation_shaded_50"
+    saturation = "elevation"
+    red = "shadedmap_r"
+    green = "shadedmap_g"
+    blue = "shadedmap_b"
+    bgcolor = "none"
+
+    @classmethod
+    def setUpClass(cls):
+        cls.use_temp_region()
+        cls.runModule("g.region", raster="elevation")
+        cls.elev_shade = "elevation_shaded_relief"
+        cls.use_temp_region()
```

Comment:
```suggestion
```

Reply:
That make sense. As I was wondering about the line, consulted the [Testing GRASS GIS code and modules](https://grass.osgeo.org/grass85/manuals/libpython/gunittest_testing.html) documentation and left the line.